ftw/contacts/tests/test_utils_get_backreferences.py

```diff
@@ -55,6 +55,7 @@ def test_do_not_append_objs_with_no_permission(self):
         self.assertEqual(
             [member_block], get_backreferences(contact, IMemberBlock))
 
+        member_block.manage_permission('View', roles=[])
```

Comment:
This does actually removes all roles with view permissions and blocks the acquisition. 
Without this change, test is failing. 

Check rolemanager.py
```
    def manage_permission(self, permission_to_manage, roles=[], acquire=0):
        """Change the settings for the given permission.

        If optional arg acquire is true, then the roles for the permission
        are acquired, in addition to the ones specified, otherwise the
        permissions are restricted to only the designated roles.
        """

```